doc/whatsnew/2/2.14/full.rst

```diff
@@ -24,6 +24,11 @@ Release date: TBA
 * Fixed a false positive for ``used-before-assignment`` when a try block returns
   but an except handler defines a name via type annotation.
 
+* ``--errors-only`` no longer explicitly enables non-error messages. This led to the unexpected
+  behavior of cancelling out prior disables.
+
+  Closes #6811
+
```

Comment:
```suggestion
* ``--errors-only`` no longer explicitly enables error messages. This led to the unexpected
  behavior of cancelling out prior disables.

  Closes #6811
```

But I guess we don't have to go into too much technical detail for user facing changes.

```suggestion
* ``--errors-only`` no longer unexpectedly enable previously disabled error messages.

  Closes #6811
```

Reply:
Ah, yes, thank you!